ddcz/forms/comments.py

```diff
@@ -0,0 +1,9 @@
+from ..models.used.discussion import Phorum
```

Comment:
...like here, I'd vote for consistency :) 

— also, please do `..models.discussions`, the used is just internal structure, all models are exported there

Reply:
Solved - it was only `from ..models import Phorum` without the discussion. :)